ChatterboxBackend/Chatterbox/bin/Debug/netcoreapp2.0/Chatterbox.runtimeconfig.dev.json

```diff
@@ -0,0 +1,9 @@
+{

```

Comment:
Oh geez!

Remove `bin` and `obj` folders from repo, and add to gitignore!

See this ignore file for the stuff that needs to be out of your repo: https://www.gitignore.io/api/visualstudio